pkg/sloop/common/db_utilities.go

```diff
@@ -57,30 +58,27 @@ func DeleteKeysWithPrefix(keyPrefix []byte, db badgerwrap.DB, deletionBatchSize
 		// deleting the keys in batch
 		if len(keysThisBatch) > 0 {
 			err, deletedKeysInThisBatch := deleteKeys(db, keysThisBatch)
-			numOfKeysToDelete += len(keysThisBatch)
 			numOfKeysDeleted += deletedKeysInThisBatch
 			if err != nil {
 				return err, numOfKeysDeleted, numOfKeysToDelete
 			}
 		}
-
-		if len(keysThisBatch) < deletionBatchSize {
-			keysLeftToDelete = false
-		}
 	}
-
 	return nil, numOfKeysDeleted, numOfKeysToDelete
-
 }
 
-func GetTotalKeyCount(db badgerwrap.DB) uint64 {
+// returns the number of keys in DB with given prefix. If prefix is not provided it gives count of all keys
+func GetTotalKeyCount(db badgerwrap.DB, keyPrefix []byte) uint64 {
```

Comment:
for keyPrefix, I see some use string, some use []byte, we can just use one form and apply it everywhere.

Reply:
I have kept string because its easier to view when debugging and change it to byte[] only when passing to the function which deletes/drops

Reply:
Can we change  keyPrefix []byte to  keyPrefix String ? From the code maintenance perspective,  we want to keep it consistent. When we call a library, we don't want to always think when to pass string and when to pass []byte, especially this is open sourced  project.